src/gamepad/controller.cpp

```diff
@@ -15,6 +20,29 @@ void Controller::updateButton(pros::controller_digital_e_t button_id) {
     }
 }
 
+void Controller::updateScreen() {
+    if (pros::millis() - this->last_print_time < 50)
+        return;
+
+    for (int i = 1; i < 3; i++) {
```

Comment:
```suggestion
    for (int i = 1; i < 4; i++) {
```
Off by one error, the loop will never check the `last_printed_line` right now.